Description/tr-TR.md

```diff
@@ -0,0 +1,89 @@
+# Türkçe
+
+Performans ve grafik optimizasyonlarına odaklanan Minecraft için Fabric tabanlı bir mod paketidir.
+
+[![Mevcut inceleme](https://img.youtube.com/vi/Vj7S5_4Rkfg/maxresdefault.jpg)](https://www.youtube.com/watch?v=Vj7S5_4Rkfg)
+
+### Performans
+
+Fabulously Optimized, Sodyum ve Lityum gibi [modern performans modlarının][1] eklenmesi nedeniyle genellikle OptiFine'dan daha hızlıdır.
+
+### OptiFine benzerliği
+
+Fabulously Optimized [neredeyse tüm OptiFine özelliklerini destekler][2], shader, bağlı dokular ve yakınlaştırma dahil. [Ücretsiz bir pelerin alın!][3]
+
+### Küçük değişiklikler
+
+Fabulously Optimized artırılmış sohbet geçmişi, pencereli tam ekran modu ve en yaygın sorunlara yönelik hata düzeltmeleri gibi vanilya hissini bozmayan [küçük değişiklikler içerir.][4]
+
+### Güncel
+
+Fabulously Optimized [hızlı güncellemeler alır][5] ve daha yeni Minecraft sürümlerine yükseltilen ilk mod paketlerinden biridir.
+
+### Her yerde çalışır
+
+Fabulously Optimized herhangi bir dünya veya sunucuda çalışmak üzere tasarlanmıştır, [4 başlatıcıda çalışır, 3 sitede barındırılır][6] ve [birçok dili destekler.][7].
+
+### Açık geliştirme
+
+Fabulously Optimized [açık kaynaklıdır][8] ve modlarının çoğu da öyle. Herkes için daha iyi bir kullanıcı deneyimi yaratmak için mod geliştiricileriyle işbirliği yaparak topluluğun fikirleriyle geliştirilmiştir.
+
+### Fabric üzerine kurulu
+
+Forge ile karşılaştırıldığında, Fabric hızlı bir şekilde güncellenir ve yüklenir, büyük kütüphaneler içermez ve yine de harika mod desteğine sahiptir. [(Evet, Quilt düşünülüyor.)][9]
+
+### Yardımsever topluluk
+
+Paketi tartışmayı ve birbirlerine yardım etmeyi seven, büyüyen arkadaş canlısı bir üye topluluğumuz var. [Bize katılın!][10]
+
+## SSS
+
+> **S**: Nasıl kurabilirim? Güncelleyebilirim? Mod ekleyebilirim? Bir şey yapabilirim?
+> 
+> **C**: [Cevaplar vikimizde.][11]
+
+
+> **S**: Bir şey bozuk! Bir şeyler eklemelisin!
+> 
+> **C**: [GitHub'ımıza gönderin.][8]
+
+
+> **S**: İçinde hangi modlar var? 
+> 
+> **C**: Herhangi bir sürüme tıklayın ve aşağı kaydırın [veya karşılaştırma tablosuna bakın][12].
+
+
+> **S**: Bunu sevdim. Nasıl katkı sağlayabilirim?
```

Comment:
Alternatif olarak "Bunu beğendim" olabilir, love and like farkını belirtmek istedim.

Reply:
Evet doğru beğendim yaptım.